gulpfile.js

```diff
@@ -39,7 +39,11 @@ gulp.task('release-specs', ['prepare-cesium'], function() {
     return build(specJSName, glob.sync(testGlob), true);
 });
 
-gulp.task('release', ['release-specs']);
+gulp.task('make-schema', function(done) {
+    child_exec('npm run make-schema', undefined, done);
```

Comment:
This is a little weird.  Usually npm calls gulp, not the other way around.  Gulp is a normal node environment, so you could package `gen-schema` as a CommonJS module, require it in, and call it directly from gulp.